### ISSUE TITLE
Idea: explore button instead of CSV button

### ISSUE THREAD

**Issue by Aariq** [uace-azmet/azmet-qa-dashboard#10]
Instead of a CSV button to download the data, a magnifying glass that when clicked expands a pop-up of the raw data.  Data would be scrollable with the initial focus on the first row that failed the validation. Failed rows could be highlighted.  I'm not sure how possible this is, but might be do-able with [`reactable`](https://glin.github.io/reactable/index.html)

**Comment by jeremylweiss** [comment]
Yes! I've been thinking along similar lines. I've found downloading several CSVs for just one or a few lines of station information to be cumbersome in the sense of having to do a few clicks just to get to the information. It'd be quicker to have a table presentation of the data appear in the browser with just one click. It also might be good to `dplyr::select` the columns so that just desired metadata and validation-error variables appear. So, a smaller, more quickly scannable table.
